### DISCUSSION TITLE
Is it possible to set brightness per LED?

### DISCUSSION THREAD

**Discussion by andrei-cb** [hyperion-project/hyperion.ng#1628] [Q&A] [answered]
Hi,

My monitor is set at an angle to the wall behind it (at about 15°). Is it possible to set the brightness of some LEDs to a higher value in order to compensate for the distance to the wall? I can only set the brightness of all LEDs.

Couldn't find any option for this in the WebUI.

**Comment by Lord-Grey** [comment, marked as answer]
Hi
Try to create an additional profile under Image Processing for the respective LEDs and define a different Brightness.
For Index you define the LED number the profile is applicable for, e.g. 0-9 for the first 10 LEDs

![image](https://github.com/hyperion-project/hyperion.ng/assets/48840279/1f0d3769-df85-44e7-9af8-0372d703261b)

**Reply by andrei-cb** [reply]
that did it, thanks!
